test/integration_test.go

```diff
@@ -13,20 +19,21 @@ import (
 	clioptions "k8s.io/cli-runtime/pkg/genericclioptions"
 	client "k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/tools/clientcmd"
-	"os"
-	"strings"
-	"testing"
-	"time"
+)
+
+var (
+	// commonLabels is a set of common labels added to each object created by this integration test, which allows us
+	// to do a proper cleanup and distinguish them from the default object created on cluster init.
+	commonLabels = labels.Set{
+		"app.kubernetes.io/name": "who-can",
```

Comment:
k delete ns,crd,role,rolebinding,clusterrole,clusterrolebindings -l app.kubernetes.io/name=who-can

Reply:
would it make sense to add that command as a comment in the code?

Reply:
Indeed. Can be useful for everyone trying to rerun the integration tests locally.